src/client/pages/OfferNew/Checkout/InsuranceSummary.tsx

```diff
@@ -226,16 +227,26 @@ const getDetails = (
     ])
   }
 
+  const getHouseHoldSizeValue = (
+    householdSize: number,
+    textKeys: TextKeyMap,
+  ) => {
+    if (householdSize === 1) {
+      return textKeys.CHECKOUT_DETAILS_SINGLE_PERSON()
+    }
+    if (householdSize > 1) {
+      return textKeys.CHECKOUT_DETAILS_PERSONS_VALUE({
+        VALUE: householdSize,
+      })
+    }
+    return '-' // or should we just throw an error?
```

Comment:
i do think it makes more sense to throw an error, otherwise we'll never know if this actually happens

Reply:
I agree! Will fix that 👍